61.md

```diff
@@ -0,0 +1,76 @@
+NIP-61
+======
+
+Unbound List
```

Comment:
Since the list order is not defined, should this be renamed to **Event Sets**?

I understand why `Unbound` is there, but branding lessons tell me that we shouldn't name things based on "not being something else" :)